application/lib/configurationElements/ConfigurationTextScreenState.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:application/elements/TextElement.dart';
+import 'package:application/model/ConfigState.dart';
+import 'package:application/view/widgets/RoundedTextField.dart';
+import 'package:flutter/material.dart';
+
+class ConfigurationTextScreenState extends ConfigState {
+  ConfigurationTextScreenState({this.element});
+
+  final TextElement element;
+  final RoundedTextField widgetName =
+      RoundedTextField('textNameField', 'Text name');
```

Comment:
Jeg tænker det er indforstået det er et tekst felt, derfor om det ikke bare kunne rettes til 'Name'?
```suggestion
      RoundedTextField('textNameField', 'Name');
```